packages/cmf/src/components/Saga/Saga.saga.test.js

```diff
@@ -0,0 +1,27 @@
+import { sagaList, actions } from './Saga.saga';
+
+describe('Saga component actions', () => {
+	function saga(attribute) {
+		return attribute;
+	}
+
+	it('should add the saga to the list', () => {
+		// given
+		const attribute = 10;
+		const sagaId = 'sagaId';
+		// when
+		actions.startSaga(sagaId, saga, attribute);
+		// then
+		expect(sagaList[sagaId]).toBe(saga);
+	});
+
+	it('should remove the saga from the list', () => {
+		// given
+		const sagaId = 'sagaId';
+		expect(sagaList[sagaId]).toBe(saga);
```

Comment:
Having those 2 tests being dependant is not a good idea.
Add an after each callback that clears the sagaList and init the sagaList when needed 🙏